x-pack/filebeat/processors/decode_cef/cef/cef_test.go

```diff
@@ -401,6 +403,26 @@ func TestEventUnpack(t *testing.T) {
 			"key2": UndocumentedField("a"),
 		}, e.Extensions)
 	})
+
+	t.Run("truncatedHeader", func(t *testing.T) {
+		var e Event
+		err := e.Unpack(truncatedHeader)
+		assert.Equal(t, errUnexpectedEndOfEvent, err)
```

Comment:
Would it be possible to make this error more descriptive based on the machine state? Like have the error indicate that the header is incomplete.

Reply:
I have added a second error in the case that there is a failure on the first round and there are no extensions.